Library/Homebrew/test/cask/cmd/uninstall_spec.rb

```diff
@@ -90,6 +90,31 @@
     expect(cask).not_to be_installed
   end
 
+  context "when Casks use script path with `~` as `HOME`" do
+    let(:dir) { TEST_TMPDIR }
```

Comment:
```suggestion
    let(:dir) { mktpmdir }
```

Also, maybe call this `home_dir`.

Reply:
Also updated in amended commit.